android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -233,6 +234,7 @@ public void onBindViewHolder(final SessionViewHolder holder, final int position)
         } else {
             holder.sessionBookmarkIcon.setImageResource(R.drawable.ic_bookmark_border_white_24dp);
         }
+        DrawableCompat.setTint(holder.sessionBookmarkIcon.getDrawable(), ContextCompat.getColor(context, Integer.valueOf(session.getTrack().getFontColor())));
```

Comment:
This is wrong. It is supposed to work like this :- 

ContextCompat.getColor(context, R.color.white);

And you are using it this way :- 

ContextCompat.getColor(context, Integer.valueOf("#ffffff"));

This will unanimously crash always